styleguide/source/_patterns/05-pages/rules-of-court.json

```diff
@@ -116,6 +116,7 @@
     ]
   },
   "mainContent": {
+    "headerIndent": true,
```

Comment:
If you move this property into the `richText` data structure (i.e. `mainContent.contents` for this snippet) does that make this work?